src/require-assert-message.spec.ts

```diff
@@ -0,0 +1,78 @@
+// require-assert-message.spec.ts
+
+/*
+ * Copyright (c) 2023 Check Digit, LLC
+ *
+ * This code is licensed under the MIT license (see LICENSE.txt for details).
+ */
+
+import { RuleTester } from 'eslint';
+
+import rule from './require-assert-message';
+
+describe('no-uuid', () => {
+  const ruleTester = new RuleTester({ parserOptions: { ecmaVersion: 2020, sourceType: 'module' } });
+  ruleTester.run('require-assert-message', rule, {
+    valid: [
+      {
+        code: `import { strict as assert } from 'node:assert';
+                       assert.ok(statusCode === StatusCodes.OK, 'Failed to get data.');`,
```

Comment:
the indenting in the test code is weird, should be two spaces and line up correctly.